lib/config.js

```diff
@@ -4,7 +4,11 @@ Config = {
     host: 'shopstyle.atlassian.net',
     user: 'shopstylebot',
     verifiedStatusName: 'Verified on QA',
-    mergedStatusName: 'Merged'
+    mergedStatusName: 'Merged',
+    toDoStatusName: 'To Do',
+    inProgressStatusName: 'In Progress',
```

Comment:
I added some extra ones here in case we needed to use them later, which I'm 90% sure we will..